imports/plugins/core/payments/server/no-meteor/resolvers/Query/paymentMethods.js

```diff
@@ -0,0 +1,5 @@
+import { paymentMethods } from "/imports/plugins/core/core/server/no-meteor/pluginRegistration";
+
+export default async function () {
+  return Array.from(paymentMethods);
```

Comment:
Same as my comment on `availablePaymentMethods`, this should be moved to a separate query function. Also, you are missing JSdoc and the new query function will need a permissions check because getting the full list is intended only for admins. The permissions check will look like this:

```js
const { userHasPermission } = context;
if (!userHasPermission(["owner", "admin"], shopId)) {
  throw new ReactionError("access-denied", "Access denied");
}
```